blenderproc/python/writer/GifWriterUtility.py

```diff
@@ -0,0 +1,210 @@
+from typing import Dict, List, Union
+from PIL import Image
+import glob
+import os
+import tempfile
+
+import bpy
+import numpy as np
+import matplotlib.pyplot as plt
+
+try:
+    from visHdf5Files import vis_data
```

Comment:
This nearly always throws an exception. Why even try to import it this way?

Reply:
Right, I have changed it.